src/stream/src/task/mod.rs

```diff
@@ -135,32 +135,42 @@ impl SharedContext {
     /// with the configured permits.
     fn get_or_insert_channels(
         &self,
-        ids: UpDownActorIds,
+        actor_ids: UpDownActorIds,
+        fragment_ids: UpDownFragmentIds,
```

Comment:
What about asserting on the `fragment_ids` identical to those in channel if exists?

Reply:
But it requires additional storage of fragment ids. I tend not to do that. 🤪